doc/OutlierFiltersFamilies.md

```diff
@@ -0,0 +1,377 @@
+
+### Other Family <a name="otherfamilyhead"></a>
+
+This family of function regroups outlier filters that can't be categorized. This is due to their behavior charging drastically (Student, Sparse ICP), dependent of a value and k (RMT, BiDistance) or a different algorithm that doesn't rely on functions (RANSAC). Due to the diversity of those functions a plot for the family can't be provided.
+
+The outlier filters RANSAC, RMT and BiDistance requires a problem in order to be computed. Therefore, the algorithm was applied to a linear regression problem on a cloud of points presenting 17% of outliers.
+
+#### Student <a name="studenthead"></a>
+
+Equations:
+![eq student](images/eq_student.JPG)
+
+Plots:
+![student fourk](images/student_fourk.png)
+
+Original reference:
+G.Agamennoni, S.Fontana, R.Y. Siegwart, and D.G. Sorrenti, "Point Clouds Registration with Probabilistic Data Association", IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS), pp.4092-4098, 2016.
+
+#### Sparse ICP <a name="sparsehead"></a>
+
+Sparse ICP is an outlier function based on the Lp norm therefore it uses a tuning parameter p. In the plots the parameter is noted k so that it can't be mistaken for the result of the cost function p(e,k). This function is only applicable for values of k in the range [0,1]. It is relevant to point out that Sparse ICP for k = 1 is equal to L1.
+
+Equations:
+![eq sparse](images/eq_sparse.JPG)
+
+Plots:
+![sparseICP fourk](images/sparseicp_fourk.png)
+
+Original reference:
+S.Bouaziz, A.Tagliasacchi, and M.Pauly, "Sparse iterative closest point", Eurographics Symposium on Geometry Processing, vol.32, no.5, pp. 113-123, 2013.
+
+#### EM-ICP <a name="emicphead"></a>
+
+The EM-ICP algorithm presents an effect on inliers similar to the step function. This is due it being based on L1. The effect on outliers is the same as the functions in the linear or sin families. Since the outlier filtering method is a trimmed version on L1 and it shares behavior with the step and linear families, it is placed in the other family.
+
+Equations:
+![eq emicp](images/eq_emicp.JPG)
+
+Plots:
+![emicp nok](images/emicp_nok.png)
+
+Original reference:
+S.Granger, and P.Xavier, "Multi-scale EM-ICP: A Fast and Robust Approach for Surface Registration", Proceedings of the European Conference on Computer Vision, pp. 418-432.
+
+#### RANSAC <a name="ransachead"></a>
+
+The RANSAC outlier filtering algorithm or RANdom SAmple Consensus relies on a voting system that uses inliers. The first step of the algorithm is to randomly sample the cloud of points. Then a model is applied depending on the problem. This model is applied to the sampled points. After the model is computed the algorithm counts the number of values contained within a [-k,k] range from the model. These values are considered as inliers. The model with the highest number of inliers is considered as the result of the algorithm. The RANSAC algorithm uses three parameters:
```

Comment:
"Then a model" -> Then,[comma] a model
"After the model is computed the algorithm " -> After the model is computed,[comma] the algorithm

Reply:
comma added